agents/mongodb/internal/profiler/aggregator/aggregator_test.go

```diff
@@ -94,4 +95,60 @@ func TestAggregator(t *testing.T) {
 			},
 		}, *result)
 	})
+
+	t.Run("createResultInvalidUTF8", func(t *testing.T) {
+		agentID := "test-agent"
+		startPeriod := time.Now()
+		aggregator := New(startPeriod, agentID, logrus.WithField("component", "test"))
+		aggregator.Start()
+		defer aggregator.Stop()
+		ctx := context.TODO()
+		err := aggregator.Add(ctx, proto.SystemProfile{
+			NscannedObjects: 2,
+			Nreturned:       3,
+			Ns:              "collection.people",
+			Op:              "query",
+			Command:         proto.BsonD{primitive.E{Key: "find", Value: "people"}, primitive.E{Key: "filter", Value: proto.BsonD{primitive.E{Key: "name_\xff", Value: "value_\xff"}}}},
+		})
+		require.NoError(t, err)
+
+		result := aggregator.createResult(ctx)
+
+		require.Equal(t, 1, len(result.Buckets))
+		assert.True(t, utf8.ValidString(result.Buckets[0].Common.Example))
+		assert.Equal(t, report.Result{
+			Buckets: []*agentpb.MetricsBucket{
+				{
+					Common: &agentpb.MetricsBucket_Common{
+						Queryid:             result.Buckets[0].Common.Queryid,
+						Fingerprint:         "FIND people name_\377",
```

Comment:
I wonder what `\377` means there

Reply:
our mongodb profiler has own escaper, I tried to find out where it's located, but couldn't find it.

Reply:
Ok, let's not spend time on that now